JoyboyCommunity/tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "extends": "expo/tsconfig.base",
-  "compilerOptions": {},
+  "compilerOptions": {
+    "jsx": "react-jsx"

```

Comment:
`jsx` compiler option should be `react-native`, and this is defined in the `expo/tsconfig.base`, so this can be removed, this is already defined.

Reply:
dont know why still the errors were coming up on my machine so had added this, will remove them